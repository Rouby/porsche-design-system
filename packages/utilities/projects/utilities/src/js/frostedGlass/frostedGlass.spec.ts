import * as fromFrostedGlass from './';

it('should provide all exports', () => {
  expect(Object.keys(fromFrostedGlass).length).toBe(1);
});

it.each<keyof typeof fromFrostedGlass>(Object.keys(fromFrostedGlass) as (keyof typeof fromFrostedGlass)[])(
  'should contain correct values for %s',
  (item) => {
    expect(fromFrostedGlass[item]).toMatchSnapshot();
  }
);
