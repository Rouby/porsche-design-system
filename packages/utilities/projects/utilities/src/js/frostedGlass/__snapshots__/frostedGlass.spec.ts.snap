// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should contain correct values for frostedGlassStyle 1`] = `
{
  "WebkitBackdropFilter": "blur(32px)",
  "backdropFilter": "blur(32px)",
}
`;
